### PR TITLE
Divyanshu | Stage | Support of bulk entry creation

### PR DESCRIPTION
**Pull request by dvCodeWorld**
https://app.clickup.com/t/86cvhkyec

### CODE REVIEW

File: apps/web-giddh/src/app/actions/ledger/ledger.actions.ts [hunk 1]
```diff
@@ -71,6 +71,18 @@ export class LedgerActions {
                 payload: res
             }, true))));
 
+    public CreateBulkBlankLedgers$: Observable<Action> = createEffect(() => this.action$
+    .pipe(
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/actions/ledger/ledger.actions.ts:81]: Add documentation

File: apps/web-giddh/src/app/actions/ledger/ledger.actions.ts [hunk 2]
```diff
@@ -405,6 +417,25 @@ export class LedgerActions {
...
 
+    /**
+     * Used to Reset Blank Ledger
+     *
+     * @returns {CustomActions}
+     * @memberof LedgerActions
+     */
+    public ResetBlankLedger(): CustomActions {
+        return {
+            type: LEDGER.RESET_BLANK_LEDGER_REQUEST,
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/actions/ledger/ledger.actions.ts]:
```suggestion
            type: LEDGER.RESET_BLANK_LEDGER_REQUEST
```

File: apps/web-giddh/src/app/actions/ledger/ledger.actions.ts [hunk 3]
```diff
@@ -405,6 +417,25 @@ export class LedgerActions {
...
+        };
+    }
+
+    public CreateBulkBlankLedgers(model: BlankLedgerVM, accountUniqueName: string): CustomActions {
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/actions/ledger/ledger.actions.ts:446]: Add documnentation

**Review by dilpreetsinghofficial — APPROVED** [review]

File: apps/web-giddh/src/app/actions/ledger/ledger.actions.ts (hunk 1 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/actions/ledger/ledger.actions.ts:81]: Done

File: apps/web-giddh/src/app/actions/ledger/ledger.actions.ts (hunk 2 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/actions/ledger/ledger.actions.ts]: Done

File: apps/web-giddh/src/app/actions/ledger/ledger.actions.ts (hunk 3 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/actions/ledger/ledger.actions.ts:446]: Done

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**